src/vm/op_contract.rs

```diff
@@ -61,27 +61,35 @@ pub enum ContractOp {
     #[display("cnc     {0},a16{1}")]
     CnC(AssignmentType, Reg32),
```

Comment:
There is a mistake here: it must use `GlobalStateType` instead of `AssignmentType`. I know it is unrelated to your changes, but it is nice to fix it in this PR as well not to forget